Lib/fontTools/merge/tables.py

```diff
@@ -132,7 +132,7 @@ def mergeOs2FsType(lst):
 	'*': first,
 	'tableTag': equal,
 	'version': max,
-	'xAvgCharWidth': avg_int, # Apparently fontTools doesn't recalc this
+	'xAvgCharWidth': first, # Will be recalculated at the end on the merged font
```

Comment:
There is also `recalculate` but I’m not sure what it does, in case you want to check it out.

Reply:
the thing is OS/2 doesn't automatically recalculate xAvgCharWidth, client (in this case merger) need to call method explicitly, so recalculate doesn't fit here I think.

Reply:
OK.